.github/workflows/labeler.yml

```
@@ -1,8 +1,7 @@
 name: release notes
 
 on:
-  pull_request:
-    types: [opened]
+  pull_request_target:
```

I think we still want to filter just to opened events.

```suggestion
  pull_request_target:
    types: [opened]
```